testing/config/settings/local.py

```diff
@@ -0,0 +1,9 @@
+# ruff: noqa: F821
```

Comment:
Is this sufficient?

Do we need to put rdmo-plugins-radar into `INSTALLED_APPS`?